### PR TITLE
Altered TEMPLATES variable to match admin.E404 error

### PR DESCRIPTION
**Pull request by Hiyorimi**
Altered variable according to [this](https://stackoverflow.com/questions/30005127/django-admin-breaks-after-upgrading-to-1-8-1) issue.

### CODE REVIEW

File: django/example/settings.py [hunk 1]
```diff
@@ -9,7 +9,7 @@
 
 DEBUG = True
 
-ALLOWED_HOSTS = []
+ALLOWED_HOSTS = ['0.0.0.0']
```
Comment (proofit404) [inline django/example/settings.py]: What was the reason to include this line in the patch?

File: django/example/settings.py [hunk 2]
```diff
@@ -51,8 +51,13 @@
...
         "DIRS": [],
         "APP_DIRS": True,
-        "OPTIONS": {
-            "context_processors": ["django.contrib.auth.context_processors.auth"]
+        'OPTIONS': {
+            'context_processors': [
+                'django.template.context_processors.debug',
+                'django.template.context_processors.request',
+                'django.contrib.auth.context_processors.auth',
+                'django.contrib.messages.context_processors.messages',
```
Comment (proofit404) [inline django/example/settings.py]: I can't understand the exact problem from the stack overflow question.

What was broken and how it was fixed with these lines?

**Comment by Hiyorimi** [comment]
Setting up environment (with clone from master) and running code with 
```bash
✗ pip install -r requirements.txt
✗ python manage.py runserver                                                                  
Watching for file changes with StatReloader
Performing system checks...

Exception in thread django-main-thread:
Traceback (most recent call last):
.....
ERRORS:
?: (admin.E404) 'django.contrib.messages.context_processors.messages' must be enabled in DjangoTemplates (TEMPLATES) in order to use the admin application.

System check identified 1 issue (0 silenced).```

results in error. So I suggest to

**Comment by sobolevn** [comment]
Sorry, we are closed during the night. Please, see the working schedule 😆

**Comment by proofit404** [comment]
Superseded by #60.

**Pull request closed without merging**